Content.Client/Chemistry/UI/ReagentDispenserWindow.xaml

```diff
@@ -1,53 +1,123 @@
+                      xmlns:controls="clr-namespace:Content.Client.UserInterface.Controls"
+                      xmlns:customControls="clr-namespace:Content.Client.Administration.UI.CustomControls"
+                      Title="{Loc 'reagent-dispenser-bound-user-interface-title'}"
+                      MinSize="680 460">
+    <BoxContainer Orientation="Horizontal">
+        <BoxContainer Orientation="Vertical" MinWidth="170">
+            <Label Text="{Loc 'reagent-dispenser-window-amount-to-dispense-label'}" HorizontalAlignment="Center" />
+            <GridContainer Columns="3"
+                           HorizontalAlignment="Center"
+                           Margin="5">
+                <!-- FIXME: OpenLeft / OpenRight misalign the text, currently just hacked my way around it with margins -->
+                <Button Name="DispenseButton1"
+                        Access="Public"
+                        Text="1"
+                        StyleClasses="OpenBoth"
+                        SetWidth="40" />
+                <Button Name="DispenseButton5"
+                        Access="Public"
+                        Text="5"
+                        StyleClasses="OpenBoth"
+                        SetWidth="40" />
+                <Button Name="DispenseButton10"
+                        Access="Public"
+                        StyleClasses="OpenLeft"
+                        SetWidth="40">
+                    <Label Text="10"
+                           Align="Center"
+                           Margin="0 0 -3 0" />
+                </Button>
+                <Button Name="DispenseButton15"
+                        Access="Public"
+                        Text="15"
+                        StyleClasses="OpenBoth"
+                        SetWidth="40" />
+                <Button Name="DispenseButton20"
+                        Access="Public"
+                        Text="20"
+                        StyleClasses="OpenBoth"
+                        SetWidth="40" />
+                <Button Name="DispenseButton25"
+                        Access="Public"
+                        Text="25"
+                        StyleClasses="OpenBoth"
+                        SetWidth="40" />
+                <Button Name="DispenseButton30"
+                        Access="Public"
+                        StyleClasses="OpenRight"
+                        SetWidth="40">
+                    <Label Text="30"
+                           Align="Center"
+                           Margin="-3 0 0 0" />
+                </Button>
+                <Button Name="DispenseButton50"
+                        Access="Public"
+                        Text="50"
+                        StyleClasses="OpenBoth"
+                        SetWidth="40" />
+                <Button Name="DispenseButton100"
+                        Access="Public"
+                        Text="100"
+                        StyleClasses="OpenBoth"
+                        SetWidth="40" />
```

Comment:
Create a helper control that can make a grid of buttons like this instead of typing them all out manually. 

Also, none of these need to be public access I don't think, so remove that arg too.